app/models/answer.rb

```diff
@@ -1,5 +1,6 @@
 class Answer < ActiveRecord::Base
   belongs_to :question
+  validates_length_of :answer, :minimum => 1           # more than 8 characters
```

Comment:
I believe we decided this also wasn't part of the main package, so let's remove it from this pull request too.

Cf:
https://trello.com/c/IkhQgjGn/206-when-being-quizzed-you-should-be-required-to-enter-text-in-the-box-before-displaying-result